src/test/java/seedu/address/logic/parser/LocalCourseAddCommandParserTest.java

```diff
@@ -43,7 +45,8 @@ public void parse_unnecessaryWhiteSpace_success() {
                         + getSquareBracketWrappedArgument(expectedLocalCourse.getLocalCode().getValue())
                         + UNNCESSARY_WHITESPACE
                         + getSquareBracketWrappedArgument(expectedLocalCourse.getLocalName().getValue())
-                        + UNNCESSARY_WHITESPACE,
+                        + UNNCESSARY_WHITESPACE
+                        + getSquareBracketWrappedArgument(expectedLocalCourse.getLocalUnit().getValue().toString()),
```

Comment:
How about putting getLocalUnit().toString() straightaway?

Reply:
good catch, in fact, I should be using the toString() method for the other fields as well instead of using getValue